content/blog/2023/08/18/2023-08-18-summer-internship-in-jenkins-security.adoc

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Summer Internship in Jenkins security"
+tags:
+- plugins
+- security
+- jenkins
+authors:
+- andreachiera
+- kguerroudj
+- wadeck
+opengraph:
+  image: /images/post-images/2023/08/18/2023-08-18-summer-internship-in-jenkins-security.png
+---
+
+=== Context
+
+Jenkins is an open-source CI/CD solution that is extensible with a wide number of plugins that can be installed using the Jenkins plugin distribution repository or via manual installation.
+
+This extensibility is a powerful feature of Jenkins, but on the other hand, it is a critical aspect that has to be secured to avoid risks and vulnerabilities that can impact the Jenkins system.
```

Comment:
```suggestion
This extensibility is a powerful feature of Jenkins, but it is a critical aspect that has to be secured to avoid risks and vulnerabilities that can impact the Jenkins system.
```
suggestion for some wording to make the verbiage more clear